TowerForge/TowerForge/ViewControllers/MainMenuViewController.swift

```diff
@@ -14,4 +14,16 @@ class MainMenuViewController: UIViewController {
         self.navigationController?.isNavigationBarHidden = false
         AudioManager.shared.playMainMusic()
     }
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == "showPlayerStats" {
+            // Get the new view controller using segue.destination.
+            // Pass the selected object to the new view controller.
+            if let playerStatsVC = segue.destination as? PlayerStatsViewController {
+                // Set any properties of playerStatsVC here
+                // For example, pass an AchievementsDatabase instance if needed
+            }
+        }
```

Comment:
Don't seem to be doing anything here. Accidental commit?

Reply:
FIxed, thanks.